mqtt/auth/client.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Mainflux
```

Comment:
This should be an independent lib, shared between all adapters (in the future). It should wrap Things gRPC client, preceded by Redis cache query.

Reply:
We can merge this for a start, then move several libs in `pkg``